### PR TITLE
chore(connector-*): update social connector description

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Update social connector description.

## Testing

UT.

### CODE REVIEW

File: packages/connector-alipay-native/src/constant.ts [hunk 1]
```diff
@@ -33,8 +33,8 @@ export const defaultMetadata: ConnectorMetadata = {
   logo: './logo.svg',
   logoDark: null,
   description: {
-    en: 'Sign In with Alipay',
-    'zh-CN': '支付宝登录',
+    en: 'Allow your users to log in through their Alipay account.',
```
Comment (IceHe) [inline packages/connector-alipay-native/src/constant.ts:36]: should align 'log in' with 'sign in'? So do below?

**Review by IceHe — APPROVED** [review]
Overall LGTM. The descriptions of the different social connectors are very similar and a little different. Maybe align in the future.

File: packages/connector-alipay-native/src/constant.ts (hunk 1 above)
Comment (darcyYe) [reply packages/connector-alipay-native/src/constant.ts:36]: These descriptions will show when you choose a social connector to add. As a result, they should be short and slightly different from others, or users may find them dull.

Comment (darcyYe) [reply packages/connector-alipay-native/src/constant.ts]: You can find similar descriptions when you try auth0 :P

**Pull request merged**